Assets/Xbox Live/Scripts/Stats/StatBase.cs

```diff
@@ -34,6 +38,37 @@ private void Awake()
         // Ensure that a StatsManager has been created so that stats will be sync with the service as they are modified.
         var statsManager = StatsManagerComponent.Instance;
     }
+
+    void Start()
+    {
+        if (XboxLiveUserHelper.Instance.SingleUserModeEnabled)
+        {
+            this.XboxLiveUser = XboxLiveUserHelper.Instance.SingleXboxLiveUser;
+            if (this.XboxLiveUser == null || this.XboxLiveUser.User == null || !this.XboxLiveUser.User.IsSignedIn)
+            {
```

Comment:
Yeah I removed it as the check in Update would handle it.

---
In reply to: [119976344](https://github.com/Microsoft/xbox-live-unity-plugin/pull/66#discussion_r119976344) [](ancestors = 119976344)